features/step_definitions/pages_steps.rb

```diff
@@ -63,4 +63,18 @@ def static_page_path(page)
 Then(/^the "([^"]*)" page title should read "([^"]*)"$/) do |str, title|
 	visit path_to(str)
 	expect(page.title).to eq "#{title}"
-end
\ No newline at end of file
+end
+
+Then(/^I see the banners for all sponsors$/) do
+sponsors = {
```

Comment:
Fix indentation here

Reply:
I had it outside the method and pasted this and forgot about the indentation